projects/plugins/jetpack/changelog/fix-contact-form-editor-width

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Form Block: fix form button alignment
```

Comment:
```suggestion
Form Block: fix form button alignment in editor
```

The alignment seems to have been working on the front end, it was just the editor that aligning right wasn't reflected, which is what this PR fixes.